docs/src/support/README.md

```diff
@@ -1,3 +1,70 @@
 # Getting Help #
 
-Page for information around where to look for help and get support.
+## I'm having trouble logging into DataLabs ##
+
+In order to login to DataLabs you'll need to create an account using the sign up button
+from the home page. Once you've created an account, you'll need to verify your e-mail
+before being able to sign in.
+
+## I have signed in but can't see anything, what do I do now ##
```

Comment:
I think this needs a `?` at the end

Reply:
mardownlint complains if headings end in punctuation including question marks, but according to https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md that shouldn't be the case.  We might need to move to https://github.com/DavidAnson/markdownlint-cli2.